test/k8s/manifests/cnp-to-cidr-oneoneoneone.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "to-cidr-oneoneoneone"
+spec:
+  endpointSelector:
+    matchLabels:
+      {}
```

Comment:
note to self: move this to a label selector that does not select any pods :facepalm: